sub-packages/bionemo-amplify/pyproject.toml

```diff
@@ -0,0 +1,31 @@
+
+[build-system]
+requires = ["setuptools>=64", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "bionemo-amplify"
+readme = "README.md"
+description = ""
+authors = [{ name = "BioNeMo Team", email = "bionemofeedback@nvidia.com" }]
+requires-python = ">=3.10"
+license = { file = "LICENSE" }
+dynamic = ["version"]
+dependencies = [
+    # internal
+    'bionemo-core',
+    'bionemo-llm',
+    # external
```

Comment:
@pstjohn will amplify depend on `bionemo-esm2` or you want to keep them independent?

Reply:
Yup good catch, done